src/services/array-functions.js

```diff
@@ -25,10 +34,15 @@ export function filter(theArray, fnc){
 //fnc will return true or false, if true return the item
 //return null
 export function find(theArray, fnc){
+  // newArray.push(theArray.find(fnc));
   const newArray = [];
-  newArray.push(theArray.find(fnc));
-  return newArray;
-
+  for (var i = 0; i < theArray.length; i++) {
+    if (fnc(theArray[i])) {
+      newArray.push(theArray[i]);
```

Comment:
My original code had it simply returning the content (string) of the result of true ('Barney').  But the test failed and said it should be an array.